src/test-utils/dom/tabs/index.ts

```diff
@@ -75,7 +76,7 @@ export default class TabsWrapper extends ComponentWrapper<HTMLButtonElement> {
    */
   findDismissibleButtonByTabId(id: string): ButtonWrapper | null {
     return this.findComponent(
-      `.${styles['tabs-tab-link']}[data-testid="${id}"] ~ .${styles['tabs-tab-dismiss']}`,
+      `.${styles['tabs-tab-link']}[data-testid="${id}"] ~ .${styles['tabs-tab-dismiss']} .${testUtilStyles['tab-dismiss-button']}`,
```

Comment:
How about simplifying this whole chain and introduce a separate data-attribute `data-testid="awsui-tab-dismiss-button-${id}"` to find this button directly?

Reply:
Will update